modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -6,14 +6,18 @@ This module included in the following assemblies:
 [id="distr-tracing-config-otel-collector_{context}"]
 = OpenTelemetry Collector configuration options
 
-The OpenTelemetry Collector consists of three components that access telemetry data:
+The OpenTelemetry Collector consists of five types of components that access telemetry data:
 
 Receivers:: A receiver, which can be push or pull based, is how data gets into the Collector. Generally, a receiver accepts data in a specified format, translates it into the internal format, and passes it to processors and exporters defined in the applicable pipelines. By default, no receivers are configured. One or more receivers must be configured. Receivers may support one or more data sources.
 
 Processors:: Optional. Processors run through the data between it is received and exported. By default, no processors are enabled. Processors must be enabled for every data source. Not all processors support all data sources. Depending on the data source, multiple processors might be enabled. Note that the order of processors matters.
 
 Exporters:: An exporter, which can be push or pull based, is how you send data to one or more back ends or destinations. By default, no exporters are configured. One or more exporters must be configured. Exporters can support one or more data sources. Exporters might be used with their default settings, but many exporters require configuration to specify at least the destination and security settings.
 
+Connectors:: A connector connects two pipelines: It consumes data as an exporter at the end of one pipeline and emits data as a receiver at the start of another pipeline. It can consume and emit data of the same or different data type. It can generate and emit data to summarize the consumed data, or it can merely replicate or route data.
+
+Extensions:: ...
```

Comment:
@iblancasa, I'm not sure whom to ask to add a suggestion here with a draft paragraph of description.